core/sphinx/sphinx_benchmark_test.go

```diff
@@ -28,8 +28,16 @@ import (
 func benchmarkSphinxUnwrap(b *testing.B, mynike nike.Scheme) {
 	const testPayload = "It is the stillest words that bring on the storm.  Thoughts that come on doves’ feet guide the world."
 
+	if mynike == nil {
+		panic("nike cannot be nil")
+	}
+
 	g := geo.GeometryFromUserForwardPayloadLength(mynike, len(testPayload), false, 5)
-	sphinx := NewSphinx(g)
+	sphinx := NewNIKESphinx(mynike, g)
+
+	if sphinx.nike == nil {
+		panic("nike cannot be nil")
```

Comment:
```suggestion
		panic("sphinx.nike cannot be nil")
```
(to disambiguate from above crash message)

Reply:
fixed